roles/rabbitmq/tasks/main.yml

```diff
@@ -44,5 +44,8 @@
 - name: correct rabbitmqadmin modes
   file: group=root owner=root mode=0755 path=/usr/local/bin/rabbitmqadmin
 
+- name: permit access to rabbitMQ server
+  ufw: rule=allow to_port={{ rabbitmq.port }} proto=tcp
```

Comment:
I have serious concerns on this one. This would open rabbit to the world, surely not what you're intending here.